integration-tests/src/test/java/com/datastax/oss/driver/api/core/context/LifecycleListenerIT.java

```diff
@@ -75,7 +75,7 @@ public void should_not_notify_listener_when_init_fails() {
       simulacronRule.cluster().acceptConnections();
     }
     assertThat(listener.ready).isFalse();
-    assertThat(listener.closed).isTrue();
+    ConditionChecker.checkThat(() -> listener.closed).before(1, SECONDS).becomesTrue();
   }
```

Comment:
maybe if you already changing this test you can change it:
`try (CqlSession session = newSession(listener)) {`
to 
`try (CqlSession ignored = newSession(listener)) {`?
My IDE complains that `session` is not used and proper `ignored` name for that.
It can be changed for both tests:
`should_notify_listener_of_init_and_shutdown` and `should_not_notify_listener_when_init_fails`